bin/setup

```diff
@@ -34,6 +34,7 @@ Dir.chdir APP_ROOT do
   run "cp keys/equifax_rsa.example keys/equifax_rsa"
   run "cp keys/equifax_rsa.pub.example keys/equifax_rsa.pub"
   run "cp certs/saml.crt.example certs/saml.crt"
+  run "which gpg || brew install gpg"
```

Comment:
We would want to `gpg --dearmor` and `gpg --import` the keys in docker case as well, how would that work?